pom.xml

```diff
@@ -0,0 +1,103 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <!-- Depends on Jenkins plugin POM to retrieve the default tooling for Groovy -->
+  <!-- TODO: create a new Parent POM for Pipeline libs?-->
```

Comment:
I'd guess it is required for the merge. Or maybe it needs a special "Pipeline Library Maven Plugin" so that I do not hack GMaven